src/main/kotlin/io/kuzzle/sdk/coreClasses/http/Route.kt

```diff
@@ -78,7 +79,15 @@ class Route {
                 "headers" -> headers.putAll(request.optMap("headers", KuzzleMap()))
                 "body" -> {
                     if (verb == "GET") {
-                        queryArgs.putAll(request.optMap("body", KuzzleMap()))
+                        var bodyMap = KuzzleMap()
+                        var body = request["body"]
+                        if (body != null) {
+                            if (body is RawJson) {
+                                body = JsonSerializer.deserialize(body.rawJson)
+                            }
+                            bodyMap = KuzzleMap.from(body as Map<String?, Any?>)
+                        }
+                        queryArgs.putAll(bodyMap)

```

Comment:
You did the right thing but after more thought there is more than RawJson that can be put inside the body, objects can also be put inside the body to be later serialized, so to fix this I would do what you did the other way around.

Something like this:
```suggestion
                        var body = request["body"]
                        if (body != null) {
                            if (!(body is Map<*, *>)) {
                              body = JsonSerializer.deserialize(JsonSerializer.serialize(body.rawJson))
                            }
                            queryArgs.putAll(body)
                        }
                       

```